ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlans.j2

```diff
@@ -3,6 +3,9 @@
 {%     for vlan in vlans | arista.avd.natural_sort %}
 vlan {{ vlan }}
    name {{ vlans[vlan].name }}
+{%         if vlans[vlan].state is defined and vlans[vlan].state is not none %}
```

Comment:
add test `and vlans[vlan].state == suspend`
`state active`, is the default and will not appear in the running configuration.

Reply:
Added as suggested.  Probably just need`if vlans[vlan].state is defined and vlans[vlan].state == "suspend"` instead of also including not none

Reply:
agreed!